.pre-commit-config.yaml

```diff
@@ -22,6 +22,7 @@ repos:
           - id: format-notebooks
             name: format-notebooks
             files: \.ipynb$
+            exclude: \.*internal.*$
```

Comment:
Don't we want the same formatting on these notebooks?

Reply:
Let me see if I can get it in good shape.

Reply:
nbqa with ruff is kinda tough to use. particularly the imports having to be at the top of the notebook. Is this a good rule?